index.js

```diff
@@ -65,6 +64,24 @@ NYC.prototype._createInstrumenter = function () {
   })
 }
 
+NYC.prototype._prepExcludePatterns = function (excludes) {
+  var directories = []
+  excludes = _.map(excludes, function (exclude) {
+    // Remove leading "current folder" prefix
+    if (_.startsWith(exclude, './')) {
+      exclude = exclude.slice(2)
+    }
+
+    // Allow gitignore style of directory exclusion
+    if (!_.endsWith(exclude, '/**')) {
```

Comment:
This was done to mimic behavior seen in ESLint's ignore methods. This may actually be an issue with ESLint as well.

Would you guys prefer I went with strict glob? No pattern adjustments at all?